### ISSUE TITLE
Sort by most-clicked

### ISSUE THREAD

**Issue by slondr** [floccusaddon/floccus#1361]
### Describe the feature you'd like to request

Nextcloud Bookmarks web view supports a "Click count" sort method, which I consider to be a killer app. It would be awesome if floccus supported that.

### Describe the solution you'd like

"Click count" sort method which syncs with the upstream Nextcloud Bookmarks's click counts.

### Describe alternatives you've considered

As far as I can tell, no approximation of this feature exists in the app

**Comment by marcelklehr** [comment]
> a "Click count" sort method, which I consider to be a killer app.

hah, that's nice to hear. I've never heard that anyone uses that.

**Comment by slondr** [comment]
Yeah, I actually switched recently from a custom-written web app which did "sort by click count" (as the only reason I wrote it) to nextcloud bookmarks specifically for that feature!

**Comment by sunjam** [comment]
Isn't this now supported via "Click Counts" functionality?

**Comment by slondr** [comment]
> Isn't this now supported via "Click Counts" functionality?

I don't see "clock counts" in the sort options; I'm on 5.2.5, is there a newer version that hasn't been pushed out yet?

**Comment by marcelklehr** [comment]
Yeah, the "Click counts" sort option is only available in Nextcloud Bookmarks at the moment.
